packages/graphql-serve/src/GraphbackServer.ts

```diff
@@ -92,37 +94,31 @@ export class GraphbackServer {
   }
 }
 
-export async function buildGraphbackServer(
-  context: InputModelTypeContext[],
-  data: GraphbackDataProvider,
-  serviceBuilder?: ServiceBuilder
-): Promise<GraphbackServer> {
-  const schemaGenerator = new SchemaGenerator(context);
-  const schema = schemaGenerator.generate();
+export async function buildGraphbackServer(graphbackConfigOpts?: any) {
+  const app = express();
 
-  const sub = new PubSub();
+  app.use(cors());
 
-  const service: GraphbackCRUDService = serviceBuilder
-    ? await serviceBuilder(data, sub)
-    : new CRUDService(data, sub);
+  if (!graphbackConfigOpts) graphbackConfigOpts = await getGraphbackConfig(path.join(__dirname, ".."));
```

Comment:
What is this about? Why we loading config from package? Should we have some defaults here and let people override it by config